examples/typescript/package.json

```diff
@@ -5,7 +5,7 @@
   },
   "devDependencies": {
     "@eslint/js": "^9.7.0",
-    "eslint": "^9.7.0",
+    "eslint": "9.14.0",
```

Comment:
Temporarily pinned until https://github.com/typescript-eslint/typescript-eslint/issues/10338 is resolved.